api-js/src/SDKMeta.ts

```diff
@@ -68,3 +69,30 @@ export namespace ReleaseHelpers {
     export const Earliest = (releases: Release[]) => releases[releases.length - 1];
     export const Latest = (releases: Release[]) => releases[0];
 }
+
+export interface UserAgent {
+    userAgents?: string[];
+    wrapperNames?: string[];
+}
+
+export const UserAgents: Record<string, UserAgent> = sdkUserAgents;
+
+export namespace UserAgentHelpers {
```

Comment:
This is a somewhat strange pattern, but it is what was done in this package already. Probably could just be top level functions.